pkgs/development/python-modules/entrypoints/default.nix

```diff
@@ -19,8 +20,15 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = lib.optional (!isPy3k) configparser;
 
+  # On python2 with pytest 3.9.2 (not with pytest 3.7.4) the test_bad
+  # test fails. It tests that a warning (exectly one) is thrown on a "bad"
+  # path. The pytest upgrade added some warning, resulting in two warnings
+  # being thrown.
+  # upstream: https://github.com/takluyver/entrypoints/issues/23
+  pyTestArgs = if isPy27 then "-k 'not test_bad'" else "";
```

Comment:
pyTestArgs is not a valid argument for `buildPythonPackage`. Use a let expression instead.

Reply:
You're right, that should've been a let expression. I merged a bit too quick. The way it is now should just add a `pyTestArgs` environment variable though, or does `buildPythonPackage` handle this different from `mkDerivation`? Do you want me to make a new PR to use `let` instead?